src/flexfix

```diff
@@ -47,5 +48,12 @@ for line in sys.stdin:
     line = re.sub(r'(#line \d+ ".*)_pretmp', r'\1', line)
     # Fix 'register' storage class specifier is deprecated and incompatible with C++17
     line = re.sub(r'register ', '', line)
+    # Force LexerInput/Output types to match those specified by the Flex declaration (might not be int on some platforms)
```

Comment:
Please add to the comment this was first needed on MacOS, and the flex version number.